lib/ansible/galaxy/token.py

```diff
@@ -130,6 +130,10 @@ def _read(self):
 
         display.vvv('%s %s' % (action, to_text(self.b_file)))
 
+        if not isinstance(config, dict):
```

Comment:
This will cause a warning for an empty token file (which may be created right above)
```suggestion
        if config and not isinstance(config, dict):
```